### DISCUSSION TITLE
请求大佬帮忙看看无法登录节点，网络错误：EOF 是怎么回事

### DISCUSSION THREAD

**Discussion by blackleavescat** [natfrp/wiki#184] [使用问题] [answered]
开始以为是节点与网络问题，于是购买了青铜VIP，换了多个节点
![Uploading 屏幕截图 2022-10-13 224032.png…]()
但是问题依旧存在，隧道一直请求失败然后打不开

**Comment by blackleavescat** [comment]
![屏幕截图 2022-10-13 224032](https://user-images.githubusercontent.com/115722899/195630803-ad29c4db-7f60-48e9-975a-41fc0260132a.png)

**Comment by fakeboboliu** [comment, marked as answer]
通常问题1：用户网络环境是校园网/公司网
问题解释：网管禁止使用
解决方案：**为了您的人身安全着想，不要用**。如果您**非常确定自己在做什么**，并且确定**没有任何网络安全问题**，确定自己**不会穿透任何有安全漏洞的应用**，确定**穿透时会正确使用 [访问认证](https://doc.natfrp.com/#/bestpractice/security?id=auth)** ，到设置中打开「使能TLS」

通常问题2：身处中国内地，用中国内地外节点，vice versa
问题解释：内地到其他地区网络连通性不能保证，特别是近期都在维护
解决方案：不要这样用

通常问题3：你拦了
问题解释：你拦了
解决方案：了解你的电脑

**Reply by blackleavescat** [reply]
感谢解答！
